Source/src/ui/WindowStateMachine.cpp

```diff
@@ -354,9 +354,9 @@ void Hachiko::WindowStateMachine::ShowLinkMenu()
         ImGui::OpenPopup("editIT");
         if (ImGui::BeginPopup("editIT"))
         {
-            std::string sourceName = r_state_machien->nodes[linkId / 100].name;
-            std::string targetName = r_state_machien->nodes[linkId % 100].name;
-            unsigned int it = r_state_machien->transitions[r_state_machien->FindTransitionWithTarget(sourceName, targetName)].blend;
+            std::string sourceName = r_state_machine->nodes[linkId / 100].name;
```

Comment:
Out of curiosity, why we operate with 100?

Reply:
Some internal shits of the library definition for the transation links, i can explain on detail on discord.

Reply:
Okayy, thanks!